src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs

```diff
@@ -160,6 +160,9 @@ private void ReplaceEmptyStringsWithNull(CsvProduct csvProduct)
             csvProduct.MainProductId = string.IsNullOrEmpty(csvProduct.MainProductId) ? null : csvProduct.MainProductId;
             csvProduct.PriceId = string.IsNullOrEmpty(csvProduct.PriceId) ? null : csvProduct.PriceId;
             csvProduct.PriceListId = string.IsNullOrEmpty(csvProduct.PriceListId) ? null : csvProduct.PriceListId;
+
+            var emptyReviews = csvProduct.Reviews.Where(x => string.IsNullOrEmpty(x.Content) && string.IsNullOrEmpty(x.ReviewType));
```

Comment:
Why not just to leave non-empty in single `Where` clause, without `Except`?

Reply:
fixed